paper-toast.html

```diff
@@ -83,9 +83,17 @@
         -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
         transition: transform 0.3s, opacity 0.3s;
         opacity: 0;
+        @apply(--paper-font-common-base);
+      }
+
+      :host(.vertical-align-bottom) {
         -webkit-transform: translateY(100px);
         transform: translateY(100px);
-        @apply(--paper-font-common-base);
+      }
+
+      :host(.vertical-align-top) {
```

Comment:
you could only add this new style, with a selector `:host([vertical-align="top"]`